lib/vehicle.js

```diff
@@ -484,11 +484,38 @@ Vehicle.prototype.unlock = function() {
   return util.wrap(req).then(({status}) => ({status}));
 };
 
+/**
+ * POST Vehicle.startCharge
+ *
+ * @return {Promise.<Security>} response on successful request
```

Comment:
```suggestion
 * @return {Promise.<ActionSuccess>} response on successful request
```

Probably rename the JSDoc typedef on line 452, so we can use it for both lock/unlock and charge actions